connectors/google-sheets/element-templates/google-sheets-connector.json

```diff
@@ -0,0 +1,586 @@
+        "name": "operation.worksheetName"
+      },
+      "constraints": {
+        "notEmpty": false,
+        "maxLength": 100
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "getRowByIndex",
+          "createRow",
+          "addValues"
+        ]
+      }
+    },
+    {
+      "label": "Dimension",
+      "description": "Choose what to add: column or row",
+      "group": "operationDetails",
+      "type": "Dropdown",
+      "optional": false,
+      "choices": [
+        {
+          "name": "Column",
+          "value": "COLUMNS"
+        },
+        {
+          "name": "Row",
+          "value": "ROWS"
+        }
+      ],
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operation.dimension"
+      },
+      "condition": {
+        "property": "operationType",
+        "equals": "addEmptyColumnOrRow"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "Row index",
+      "description": "Enter row index",
+      "group": "operationDetails",
+      "type": "String",
+      "feel": "optional",
+      "optional": false,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operation.index"
+      },
+      "constraints": {
+        "notEmpty": true,
+        "maxLength": 100,
+        "pattern": {
+          "value": "^([0-9]*$)|(secrets.*$)",
+          "message": "Must be a number."
+        }
+      },
+      "condition": {
+        "property": "operationType",
+        "equals": "getRowByIndex"
+      }
+    },
+    {
+      "label": "Row index",
+      "description": "Enter the index of the row to add",
+      "group": "operationDetails",
+      "type": "String",
+      "feel": "optional",
+      "optional": true,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operation.index"
+      },
+      "constraints": {
+        "notEmpty": false,
+        "maxLength": 100,
+        "pattern": {
+          "value": "^([0-9]*$)|(secrets.*$)",
+          "message": "Must contains only numbers."
+        }
+      },
+      "condition": {
+        "property": "operationType",
+        "equals": "createRow"
+      }
+    },
+    {
+      "label": "Column index",
+      "description": "Enter the index of the column",
+      "group": "operationDetails",
+      "type": "String",
+      "feel": "optional",
+      "optional": false,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operation.index"
+      },
+      "constraints": {
+        "notEmpty": true,
+        "maxLength": 100,
+        "pattern": {
+          "value": "[A-Z]+",
+          "message": "Must contains only capital letters."
+        }
+      },
+      "condition": {
+        "property": "operationType",
+        "equals": "deleteColumn"
+      }
+    },
+    {
+      "label": "Start index",
+      "description": "Enter start index (leave empty if add to the end of the sheet)",
```

Comment:
can you add here an example?

Reply:
a short example is always helpfull and a faster solution for our enduser. A good example is
image

here we are saying, what we are expecting and referencing the documentation for follow up questions

Reply:
Add link to the documentation where it'll be described